test/commands_map.ts

```diff
@@ -108,25 +101,23 @@ var commands_map: CommandValuesMap = {
         }
     },
     "carousel-scroll": {
-        client: async function () {
+        client: () => new Promise((resolve) => {
             var right_arrow = document.querySelectorAll('.scroll.next')[0] as HTMLButtonElement;
-            
+
             // Scrolling the carousel simulating touch action
             var car_items = document.querySelectorAll('.wc-carousel-item').length;
-            for(var i=0; i<car_items; i++){
-                var element = document.querySelectorAll('.wc-carousel-item')[i];
-                element.scrollIntoView();
-                await new Promise((resolve) => {
-                    setTimeout(resolve, 300);
-                }); 
+            for (var i = 0; i < car_items; i++) {
+                    var element = document.querySelectorAll('.wc-carousel-item')[i];
+                    element.scrollIntoView();
             }
-
-            return right_arrow.getAttribute('disabled') != null;
-        },
+            setTimeout(() => {
+                resolve(right_arrow.getAttribute('disabled') != null);
+            }, 500);
+        }),
```

Comment:
I think you may need to merge latest then reapply your markdown changes to isolate your changes from others in this changeset.